test/find.test.ts

```diff
@@ -59,4 +60,48 @@ describe('find', () => {
       expect(err).toEqual(expected);
     }
   });
+
+  it('should skip files equal or greater than 2GB', async () => {
+    const MAX_SIZE = 2 * 1024 * 1024 * 1024;
+
+    const fakeFS: { [key: string]: any } = {
+      projDir: {
+        isFile: () => false,
+        isDirectory: () => true,
+        size: 0,
+      },
+      'under.cpp': {
+        isFile: () => true,
+        isDirectory: () => false,
+        size: MAX_SIZE - 1,
+      },
+      'max.hpp': {
+        isFile: () => true,
+        isDirectory: () => false,
+        size: MAX_SIZE,
+      },
+      'above.hpp': {
+        isFile: () => true,
+        isDirectory: () => false,
+        size: MAX_SIZE + 1,
+      },
+    };
+
+    const fakeFsHandler: FsHandler = {
+      stat: (fullpath: fs.PathLike) => {
```

Comment:
we should avoid declaring functions inside of another functions

Reply:
I wanted it to have higher coupling since I want to prevent others from reusing it.